astoria/common/messages/astmetad.py

```diff
@@ -42,6 +42,10 @@ def init(cls, config: AstoriaConfig) -> 'Metadata':
             libc_ver="".join(platform.libc_ver()),
             kit_name=config.kit.name,
             kit_version=config.kit.version,
+            wifi_ssid=config.wifi.ssid,
+            wifi_psk=config.wifi.psk,
+            wifi_region=config.wifi.region,
```

Comment:
```suggestion
            wifi_region=config.wifi.region,
```

These already have default values of `None`, so we can leave them alone.